examples/chart/teleport-cluster/templates/auth/_config.common.tpl

```diff
@@ -0,0 +1,64 @@
+{{- define "teleport-cluster.auth.config.common" -}}
+{{- $authentication := mustMergeOverwrite .Values.authentication (default dict .Values.authenticationSecondFactor) -}}
+{{- $logLevel := (coalesce .Values.logLevel .Values.log.level "INFO") -}}
+version: v3
+teleport:
+  log:
+    severity: {{ $logLevel }}
+    output: {{ .Values.log.output }}
+    format:
+      output: {{ .Values.log.format }}
+      extra_fields: {{ .Values.log.extraFields | toJson }}
```

Comment:
The teleport block should be at the end of the document as the aws/gcp modes are setting storage, which is in `teleport` and not in `auth_service`